### ISSUE TITLE
fix 'make install'

### ISSUE THREAD

**Issue by beaumanvienna** [beaumanvienna/marley#34]
`make install` is installing a static library, which is not supposed to to happen:
```
/usr/bin/mkdir -p '/usr/local/lib'
 /usr/bin/install -c -m 644  libmednafen_marley.a '/usr/local/lib'
```

It is also outputting a to of weird stuff, see [terminal output make install.txt](https://github.com/beaumanvienna/marley/files/5251187/terminal.output.make.install.txt)

**Comment by beaumanvienna** [comment]
partially addressed with aad86ca135933cb6c5bc5b662f1582d6395bffe7

**Comment by beaumanvienna** [comment]
this commit is fixing the remaining issues: ea2576a62239392441
